lib/tasks/change_note.rake

```diff
@@ -0,0 +1,37 @@
+namespace :change_note do
+  desc "List change notes for a content ID"
+  task :list, %i[content_id] => :environment do |_, args|
+    change_history = Document
+      .find_by(content_id: args[:content_id])
+      .editions
+      .where(minor_change: false)
+      .pluck(:id, :major_change_published_at, :change_note)
+
+    change_history.each do |change|
+      puts change.join("\t")
+    end
+  end
+
+  desc "Amend a change note for an edition"
+  task :amend, %i[edition_id new_change_note email] => :environment do |_, args|
+    edition = Edition.find(args[:edition_id])
+    user_id = User.find_by(email: args[:email]).id
```

Comment:
I guess there's a small chance that we might end up calling `.id` on `nil` if for example the task is run by someone new to gov.uk who hasn't set up signon access for Whitehall yet?

Reply:
That's a good point.  I'll add a guard to ensure the user exists before we continue any further.